bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -79,6 +81,22 @@ trait UtilBijections {
   }
 
   /**
+   * Injection from java futures to twitter futures
+   */
+  implicit def twitter2JavaFuture[A](
```

Comment:
it might make sense to expose an `injection` which doesn't need a converter too, in case folks are only going `JFuture => TwitterFuture`.